testdata/simple-go/utils.go

```
@@ -0,0 +1,283 @@
+
+// GenerateID generates a random ID
+func GenerateID() string {
+	// This function calls generateRandomBytes
+	bytes := generateRandomBytes(8)
+	return hex.EncodeToString(bytes)
+}
+
+// generateRandomBytes generates random bytes
+func generateRandomBytes(length int) []byte {
+	bytes := make([]byte, length)
+	rand.Read(bytes)
```

golangci-lint (1.64.8)

Error return value of `rand.Read` is not checked

(errcheck)